src/aavrutiDJun20/array/IntersectionOfTwoArray.java

```diff
@@ -0,0 +1,74 @@
+package aavrutiDJun20.array;
+
+public class IntersectionOfTwoArray {
+	int duplicateNumberCnt = 0;
+	
+	//Store count of all the duplicate numbers from two arrays and store it in temporary array
+	double[] findIntersectionOfArray(double[] array1,double[] array2) {
+		int numberCnt = 0;
+		int arrayLength = 0;
+		
+		if(array1.length>=array2.length)
+			arrayLength = array1.length;
+		else
+			arrayLength = array2.length;
+		
+		double[] tempIntersectionArray = new double[arrayLength];
+		String[] str = new String[arrayLength];
```

Comment:
@AavrutiDJun20 , String array is not required here and you haven't used it here.
at line number 31 you have if condition, you are checking * but you didnt write a logic to place a * at any index. isn't it ? According to me line number 17,18,31 is extra.

Reply:
I got it , you have used * logic but i think String array is an overhead.